cob_twist_controller/src/limiters/limiter.cpp

```diff
@@ -159,7 +159,7 @@ KDL::JntArray LimiterAllJointPositions::enforceLimits(const KDL::JntArray& q_dot
 
     if (tolerance_surpassed)
     {
-        ROS_WARN("Tolerance surpassed: Enforcing limits FOR ALL JOINT POSITIONS with factor = %f", factor);
+        ROS_DEBUG("Tolerance surpassed: Enforcing limits FOR ALL JOINT POSITIONS with factor = %f", factor);
```

Comment:
Warnings make pretty much sense here to show the operator that the arm cannot move anymore! 

If DEBUG it might not be obvious without roslog or any other tool!